test/controllers/AccountHomeControllerSpec.scala

```diff
@@ -102,7 +102,7 @@ class AccountHomeControllerSpec extends SpecBase {
 
       when(
         mockBackEndConnector.applicationSummaries(any())
-      ) thenReturn Future
+      ) `thenReturn` Future
```

Comment:
I prefer using .thenReturn() over backticks. I've posted a comment in our slack dev channel to see what people say.